frontend/src/components/Bundle/Bundle.js

```diff
@@ -136,6 +145,20 @@ class Bundle extends React.Component<
     /** Renderer. */
     render() {
         const bundleInfo = this.state.bundleInfo;
+
+        // Below block can be used to map any errorCode to custom message for user
```

Comment:
```suggestion
        // Show custom messages based on error code
```

Reply:
We should add this entire block into the `if (!bundleInfo) {` block, for consistency with the other error handling logic in this component

Reply:
I did that earlier but decided to keep it outside the `if` block because we do get some info in `bundleInfo` variable at times and error is not triggered in those cases.

Reply:
I think this PR looks good, just update the comment as suggested and it should be good to go.

Reply:
> I did that earlier but decided to keep it outside the `if` block because we do get some info in `bundleInfo` variable at times and error is not triggered in those cases.

When would `bundleInfo` be non-null and `this.state.errorCode` be 403/404? I would have thought `bundleInfo` is supposed to be null in all those cases.

Reply:
Even in the case of 403 error we do get some content in bundleInfo. I am making the changes in the comment as per your suggestion. We can merge this one and test on dev to see if it works fine.

Reply:
> I think this PR looks good, just update the comment as suggested and it should be good to go.

Done! Please +1 if it's good to go from your side @jzwang43

Reply:
@pranavjain 
> Even in the case of 403 error we do get some content in bundleInfo

Got it, I'm just asking if you understand why that is the case -- what is in bundleInfo when we get a 403 error? I'm trying to see if this is expected behavior or another bug that we should file.

Reply:
I think there is another bug in the code. We can follow up on that one separately. I will investigate and file a separate issue. Let's close this one. Thanks!

Reply:
Ok